--- mate-volume-control/gvc-channel-bar.c.orig	2015-08-19 23:17:19.841318807 +0200
+++ mate-volume-control/gvc-channel-bar.c	2015-08-20 20:32:14.904986288 +0200
@@ -26,7 +26,7 @@
 #include <glib-object.h>
 #include <gtk/gtk.h>
 
-#include <canberra-gtk.h>
+#include <canberra-gtk2.h>
 #include <libmatemixer/matemixer.h>
 
 #include "gvc-channel-bar.h"
--- mate-volume-control/gvc-balance-bar.c.orig	2015-08-19 23:16:49.238474008 +0200
+++ mate-volume-control/gvc-balance-bar.c	2015-08-19 23:16:59.814420369 +0200
@@ -24,7 +24,7 @@
 #include <glib-object.h>
 #include <gtk/gtk.h>
 
-#include <canberra-gtk.h>
+#include <canberra-gtk2.h>
 #include <libmatemixer/matemixer.h>
 
 #include "gvc-balance-bar.h"
--- mate-volume-control/gvc-sound-theme-chooser.c.orig	2015-08-20 20:35:05.947552434 +0200
+++ mate-volume-control/gvc-sound-theme-chooser.c	2015-08-20 20:35:18.899292461 +0200
@@ -32,7 +32,7 @@
 #include <glib-object.h>
 #include <gio/gio.h>
 #include <gtk/gtk.h>
-#include <canberra-gtk.h>
+#include <canberra-gtk2.h>
 #include <libxml/tree.h>
 
 #include "gvc-sound-theme-chooser.h"
